cli/pkg/kctrl/cmd/package/init/pkg_build.go

```diff
@@ -73,11 +77,61 @@ func (b *PackageBuild) SetObjectMeta(metaObj *metav1.ObjectMeta) {
 	return
 }
 
-func (b PackageBuild) GetExport() *[]appbuild.Export {
+func (b *PackageBuild) GetExport() *[]appbuild.Export {
 	return &b.Spec.Template.Spec.Export
 }
 
 func (b *PackageBuild) SetExport(exportObj *[]appbuild.Export) {
 	b.Spec.Template.Spec.Export = *exportObj
 	return
 }
+
+func (b *PackageBuild) HasHelmTemplate() bool {
+	appSpec := b.getAppSpec()
+	if appSpec == nil || appSpec.Template == nil {
+		return false
+	}
+	appTemplates := appSpec.Template
+	for _, appTemplate := range appTemplates {
+		if appTemplate.HelmTemplate != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func (b *PackageBuild) ConfigureExportSection() {
+	fetchSource := b.GetObjectMeta().Annotations[appbuild.FetchContentAnnotationKey]
+	exportSection := *b.GetExport()
```

Comment:
```suggestion
	exportSection := b.GetExport()
```

Reply:
We won't be able to do length checks on `Export` if we make that change.
But I guess `GetExport` could return a non-pointer value

Reply:
Made this change ^